src/main/java/com/gmail/stepura/volodymyr/service/CsvToBeanParserService.java

```diff
@@ -0,0 +1,9 @@
+package com.gmail.stepura.volodymyr.service;
+
+import com.univocity.parsers.common.record.Record;
+import java.io.IOException;
+import java.util.List;
+
+public interface CsvToBeanParserService {
+    void saveBeans(List<Record> strings) throws IOException;
```

Comment:
the purpose of this interface is to save data to db - let's name it accordingly

```suggestion
    void saveEntities(List<Record> records);
```
if any IOException is possible - let's manage it in the service

Reply:
Fixed naming, removed redundant Exception